receiver/splunkenterprisereceiver/README.md

```diff
@@ -8,7 +8,9 @@ jobs.
 
 ## Configuration
 
-The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping.
+The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping. 
+
+**NOTE:** These must be set for each Splunk instance type (indexer, search head, or cluster master) from which you wish to pull metrics. At present, only one of each type is accepted, per configured receiver instance. This means, for example, that if you have three different "indexer" type instances that you would like to pull metrics from you will need to configure three different `splunkenterprise` receivers for each indexer node you wish to monitor.
 
 * `basicauth` (from [basicauthextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/basicauthextension)): A configured stanza for the basicauthextension.
```

Comment:
Unrelated: can we deprecate and remove this option give that we have `auth`?

Reply:
I suppose we can, are we planning on moving away from the basicauth extension entirely? Also a little hesitant at the moment since `auth` is not super well documented right now.

Reply:
I think it's just confusing to have two config options for the same purpose